packages/catalog-process/src/model/domain/errors.ts

```diff
@@ -9,37 +9,28 @@ import {
 import { RiskAnalysisValidationIssue, logger } from "pagopa-interop-commons";
 
 export const errorCodes = {
-  eServiceDescriptorNotFound: "0002",
-  eServiceDescriptorWithoutInterface: "0003",
-  notValidDescriptor: "0004",
-  eServiceDocumentNotFound: "0006",
-  eServiceNotFound: "0007",
-  draftDescriptorAlreadyExists: "0008",
-  eserviceCannotBeUpdatedOrDeleted: "0009",
-  eServiceDuplicate: "0010",
-  originNotCompliant: "0011",
-  attributeNotFound: "0012",
-  inconsistentDailyCalls: "0013",
-  interfaceAlreadyExists: "0014",
-  eserviceNotInDraftState: "0015",
-  eserviceNotInReceiveMode: "0016",
-  tenantNotFound: "0017",
-  tenantKindNotFound: "0018",
-  riskAnalysisValidationFailed: "0019",
```

Comment:
Adjust the numbering, and remove `eserviceCannotBeUpdatedOrDeleted`, now the `updateEService` and `deleteEService` are throwing `eserviceNotInDraftState`.